source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
 Configuration
 -------------
 
-Minimal "sane" config:
+The configuration of Unbound can be a little tricky due to the extensive array of configurable options. Below we will go through a basic, recommended config, but feel free to add and experiment with options as you need them. Also feel free to remove lines that are commented out (uncommenting bu removing the "#") if they are not nesecary in your setup.
+
+For the configuration step, we will assume that your system has a Unbound installed and it is available to the entire system (so the :command:`make install` step during installation). 
```

Comment:
Perhaps replace this sentence with: Note that the instructions in this page assume that Unbound is already installed. I'm unsure how wise it is to reference `make install` as some/most users will use packages rather than installing from sources?